test/NHSD.BuyingCatalogue.Documents.API.Integration.Tests/Support/IntegrationSetUpFixture.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Threading.Tasks;
+using TechTalk.SpecFlow;
+
+namespace NHSD.BuyingCatalogue.Documents.API.IntegrationTests.Support
+{
+    internal class IntegrationSetUpFixture
+    {
+        [Binding]
+        public class IntegrationSetupFixture
+        {
+            private readonly AzureBlobStorageScenarioContext _scenarioContext;
+            public IntegrationSetupFixture(AzureBlobStorageScenarioContext scenarioContext)
+            {
+                this._scenarioContext = scenarioContext;
+            }
+
+            [AfterScenario()]
```

Comment:
You can omit the brackets from the attribute.

Reply:
👍